postgis/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM timescale/timescaledb:0.0.11-beta
+FROM timescale/timescaledb:latest
```

Comment:
Another issue is that you typically have a latest tag cached in your local Docker. If you want to use latest tag you should make sure to always fetch the newest "latest" tag first.